lib/devise/omniauth.rb

```diff
@@ -8,8 +8,8 @@
   raise
 end
 
-unless OmniAuth::VERSION =~ /^1\./
-  raise "You are using an old OmniAuth version, please ensure you have 1.0.0.pr2 version or later installed."
+if Gem::Version.new(OmniAuth::VERSION) < Gem::Version.new('1.0.0')
+  raise "You are using an old OmniAuth version, please ensure you have 1.0.0 version or later installed."
```

Comment:
Should we use `gem 'omniauth', '>= 1.0.0.pr2'` here?

Reply:
Do you mean use the `gem` command instead of the manual version check here? That's probably best yes, I'll update.

Or is it about the `pr2` version? I thought we could just set to 1.0+ stable and don't allow the pr/rc ones anymore.

There was very little change between the two https://github.com/omniauth/omniauth/compare/v1.0.0.pr2...v1.0.0, and hopefully there's no one else running those pre-stable versions.